tsconfig.json

```diff
@@ -6,11 +6,7 @@
 		"module": "es2015",
 
 		// JSON files are declared as CJS modules in TS
-		"allowSyntheticDefaultImports": true,
-		"lib": [
-			"dom",
-			"dom.iterable"
-		]
+		"allowSyntheticDefaultImports": true
```

Comment:
I would remove this. It casues weird problems sometimes. Better to be explicit about what the module types are.

Reply:
This appears to be the only way to import JSON files in TypeScript (without creating a custom `declare module`). For whatever reason it parses the file correctly but then it defines internally it as `export =` instead of `export default`

Reply:
I mean `import`ing

Reply:
Gotta love all those weird TS limitations...

Reply:
Probably safer to restore a `global.d.ts` file with the `declare module`: https://github.com/vadimdemedes/dom-chef/pull/54/commits/0b33201eafb72ab1281e545e74ec514431104918#diff-46d5f7748f620eb038554954b14a0b16

Maybe it's just:

```
declare module 'svg-tag-names' { export default [] as string[]; }
```

Semi-related: https://github.com/microsoft/TypeScript/issues/15146

Reply:
I think TypeScript's biggest problem is non-thorough documentation. I couldn't find any mention of JSON imports in it.

Reply:
Reported https://github.com/microsoft/TypeScript/issues/37623 and dropped `allowSyntheticDefaultImports` https://github.com/vadimdemedes/dom-chef/pull/56/commits/18c15c821e2d2e7557364e982959a5f2c9f05b56